ksql-common/src/main/java/io/confluent/ksql/schema/ksql/Column.java

```diff
@@ -95,25 +113,45 @@ public ColumnName name() {
   /**
    * @return the type of the field.
    */
+  @Override
   public SqlType type() {
     return type;
   }
 
   /**
    * @return the column reference
    */
+  @Override
   public ColumnRef ref() {
     return ref;
   }
 
+  /**
+   * @return the column namespace.
+   */
+  public Namespace namespace() {
+    return namespace;
+  }
+
+  /**
+   * @return index of the column <i>within the namespace</i>.
```

Comment:
ah this is clever - I wanted to make this change earlier but got caught up trying to figure out what a good index would be for meta/key fields.